### DISCUSSION TITLE
Idea: Easier clearing in/out marks with right click or shift press

### DISCUSSION THREAD

**Discussion by jakweg** [remotion-dev/remotion#1221] [Ideas]
I'm talking about these buttons:
![image](https://user-images.githubusercontent.com/20194971/184135276-2d2ab3ce-2e96-416e-a67f-18528e065e89.png)

When using I always struggle with easily clearing _out_ mark. I first need to pause the video, then double click mark out button, then resume. There is a shortcut X to clear both, but let's say I want to only clear the _out_ one. 

Therefore I suggest adding additional action on right click: clear mark. 
And also for keyboard users: **I** marks _in_, **shift+I** can clear it, **O** marks _out_, **shift+O** to clear it

**Comment by JonnyBurger** [comment]
Hi Jakub!

Sorry for the delay - nonetheless a valid idea! I'll transfer this to an issue.
